services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -203,6 +205,10 @@ async def list_files(  # noqa C901
                     for node_id, node_data in proj_data.workbench.items()
                 }
 
+        # expanding directories
+        for entry in delayed_expands:
+            data.extend(await entry)
+
         # FIXME: artifically fills ['project_name', 'node_name', 'file_id', 'raw_file_path', 'display_file_path']
```

Comment:
Not being the original author of this, I cannot comment on why this is here.
What I can say is that it has to stay, since I don't see any other way to put that information inside.

Reply:
yes this is some old crap that needs to go in sync with the frontend.

Reply:
Let me toss away the fix, since it will stay there